tests/Core/Tokenizer/ScopeOwnerTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Tests that scope owner (scope_condition, conditions) is set correctly
+ *
+ * @author  Dan Wallis <dan@wallis.nz>
+ * @license https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
+ */
+
+namespace PHP_CodeSniffer\Tests\Core\Tokenizer;
+
+use PHP_CodeSniffer\Tests\Core\AbstractMethodUnitTest;
+
+final class ScopeOwnerTest extends AbstractMethodUnitTest
+{
+
+
+    /**
+     * Test that a basic 'if' condition gets scope opener/closer set as expected.
+     *
+     * @param string $testMarker     The comment which prefaces the target token in the test file.
+     * @param int    $conditionWidth How many tokens wide the 'if' condition should be.
+     *
+     * @dataProvider dataIfCondition
+     * @covers       PHP_CodeSniffer\Tokenizers\PHP::tokenize
```

Comment:
Is this `@covers` tag correct ? The change is in the `Tokenizer::recurseScopeMap()` method ?